tests/resolvers/post_query/postsByOrganization.spec.js

```diff
@@ -0,0 +1,590 @@
+const axios = require('axios');
+const { URL } = require('../../../constants');
+const getToken = require('../../functions/getToken');
+const shortid = require('shortid');
+
+const database = require('../../../db');
+const getUserId = require('../../functions/getUserId');
+const Organization = require('../../../lib/models/Organization');
+const Post = require('../../../lib/models/Post');
+const Comment = require('../../../lib/models/Comment');
+const User = require('../../../lib/models/User');
+
+// let mainToken;
+// let secondToken;
+// let thirdToken;
```

Comment:
if these variables aren't being used, it would be best to remove them.

Reply:
I used them to initialize the seed data using the api then I faced a problem on adding imageUrl so used the database models directly and I'm not sure whether to leave it this way or change it so I just commented them if the database approach is ok with you i will remove them.

Reply:
also the sorting problem I just solved it in place but I think a better way is to add a likeCount and commentCount property inside the user document to keep track of them directly from the database but  again I'm just waiting for feed back from you guys.

Reply:
Thanks, use the count properties.

Reply:
> I used them to initialize the seed data using the api then I faced a problem on adding imageUrl so instead I used the database models directly and I'm not sure whether to leave it this way or change it so I just commented them if the database approach is ok with you i will remove them.

What do you suggest as the best long term approach?

Reply:
> > I used them to initialize the seed data using the api then I faced a problem on adding imageUrl so instead I used the database models directly and I'm not sure whether to leave it this way or change it so I just commented them if the database approach is ok with you i will remove them.
> 
> What do you suggest as the best long term approach?

I think using the database directly is better for performance ( better than using the whole api ) of tests which is beneficial with the increasing of tests added to the system, also the database approach solves the the problem I faced of adding imageUrl field without needing to pass unnecessary input validation etc..

Reply:
OK, please proceed. We need to fix this as many of our pushes to the repo are intermittently failing.